app/src/components/trace/TraceTree.tsx

```diff
@@ -140,9 +140,9 @@ function SpanNodeWrap(props: PropsWithChildren<{ isSelected: boolean }>) {
  * The line that connects the parent node to the child node edge
  */
 function SpanTreeEdgeConnector({
-  statusCode,
+  propagatedStatusCode,
 }: {
-  statusCode: SpanStatusCodeType;
+  propagatedStatusCode: SpanStatusCodeType;
```

Comment:
same as above, leave the contract as is, make the shape conform rather than making the component bend to the will of the API request.

Reply:
addressed in [c4f88f4](https://github.com/Arize-ai/phoenix/pull/1824/commits/c4f88f45f6b6b15167a7284e8484e93340e4b1ac)